src/main/java/dao/DBCreator.java

```diff
@@ -0,0 +1,52 @@
+package dao;
+
+import org.sqlite.SQLiteDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DBCreator {
```

Comment:
Can you rename it to `DBConfiguration` or `DataSourceConfiguration`?

Reply:
Done